codegen.yml

```diff
@@ -9,4 +9,8 @@ generates:
       - "typescript-resolvers"
       - "typescript-react-apollo"
     config:
-      withHooks: true
\ No newline at end of file
+      withHooks: true
+      withHOC: true
```

Comment:
Why are we still using apollo HOCs? I remember right I needed to add this line every time I recreated graphql types.

Reply:
Some pages still use the `withGetApp` HOC instead of `useGetAppQuery` hook.